.github/workflows/lint_test.yml

```diff
@@ -1,13 +1,19 @@
-name: Lint
+name: Lint & Test
```

Comment:
I prefer to split the lint and test into two workflow files, but it's also ok for me